contributing/development/configuring_an_ide/neovim.rst

```diff
@@ -0,0 +1,52 @@
+.. _doc_configuring_an_ide_rider:
+
+Neovim
+======
+
+`Neovim <https://neovim.io/>`__ is a hyper-extensible vim-based editor that is fully free to use and open-source under the Apache 2.0 license.
+On it's own, it is as simple as a plain-text editor, but with plugins it can be extended to be a fully fledged IDE of your own design.
```

Comment:
```suggestion
On its own, it is as simple as a plain-text editor, but with plugins it can be extended to be a fully fledged IDE of your own design.
```